docs/cudf/source/basics/nested.rst

```diff
@@ -0,0 +1,35 @@
+Nested
+======
+
+
+Struct and List datatypes
+----------------
+
+cuDF supports arbitrarily deep nested lists. Such as ``list(list(int))``, even list of structs or structs of lists
+
+cuDF also supports arbitrary fields for structs - that is, it is possible to have a struct with any number of fields and any number of types that cuDF supports, even a struct of structs
+
+Structs should be made up by same type of datatype or cuDF will produce an error - Example below
+    
+.. code-block:: python
+    
+    >>> df = cudf.Series(
+    >>> [{'a':'dog', 'b':'cat', 'c':'astronomy'},
+    >>> {'a':'fish', 'b':'gerbil', 'c':7}]
+    >>> )
+    >>> df
```

Comment:
Is the error produced at the time of construction, or when attempting to print the `df`?